### PR TITLE
Add circle config and rubocop

### PR DESCRIPTION
**Pull request by aaron-collier**
Adds circle and rubocop. This is a relatively simple repo, we may want to add tests later, but for now we should at least rubocop it.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      - save_cache:
+          paths:
+            - ./vendor/bundle
+          key: v1-dependencies-{{ checksum "Gemfile.lock" }}
+
+      # run tests!
+      - run:
+          name: Check styles using rubocop
+          command: bundle exec rubocop
```
Comment (mjgiarlo) [inline .circleci/config.yml:41]: Should we add rspec and coverage here as well?

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,423 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2019-09-25 10:06:45 -0700 using RuboCop version 0.74.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
```
Comment (mjgiarlo) [inline .rubocop_todo.yml]: Are you cool with running auto-correct and then re-generating this file? That way we can knock out trivial exclusions.

File: Gemfile [hunk 3]
```diff
@@ -6,3 +6,4 @@ gem 'dlss-capistrano'
 gem 'capistrano-bundler'
 gem 'capistrano-rvm'
 gem 'rake'
+gem 'rubocop'
```
Comment (mjgiarlo) [inline Gemfile]: Can you pin this to `~> '0.74.0'`?

**Review by mjgiarlo — CHANGES_REQUESTED** [review]
Have some questions

File: .rubocop.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```
Comment (mjgiarlo) [inline .rubocop.yml:1]: Do we need any of this stuff here? https://github.com/sul-dlss/argo/blob/master/.rubocop.yml#L2-L13

File: .rubocop_todo.yml (hunk 2 above)
Comment (aaron-collier) [reply .rubocop_todo.yml]: Done.

File: .rubocop.yml (hunk 4 above)
Comment (aaron-collier) [reply .rubocop.yml:1]: We don't have any of those paths... Set the version though.

File: .circleci/config.yml (hunk 1 above)
Comment (ndushay) [reply .circleci/config.yml:41]: only after there are actual tests!

Comment (mjgiarlo) [reply .circleci/config.yml:41]: no tests, nope.

**Review by mjgiarlo — APPROVED** [review]

File: Gemfile [hunk 5]
```diff
@@ -1,8 +1,11 @@
...
 source 'https://rubygems.org'
 
 gem 'bundler'
 gem 'capistrano', '~> 3.11', require: false
-gem 'dlss-capistrano'
 gem 'capistrano-bundler'
 gem 'capistrano-rvm'
+gem 'dlss-capistrano'
 gem 'rake'
+gem 'rubocop', '~> 0.74.0'
```
Comment (ndushay) [inline Gemfile:11]: hooray for alphabetical!

**Review by ndushay — CHANGES_REQUESTED** [review]
LGTM - it's a start, anyhoo.

Could you please add the circleci badge to the README?

**Review by ndushay — COMMENTED** [review]
oops - you did. d'oh

**Pull request merged**